app/controllers/api/v1/medias_controller.rb

```diff
@@ -212,13 +212,14 @@ def create
        ]}
      '
       def index
-        @medias = Media.where(account_id: params[:account_id], 
-                              asset_id: params[:asset_id], 
-                              title: params[:title], 
-                              duration: params[:min_duration].to_i..params[:max_duration].to_i)
-        if params[:sort == 'True')
-          @medias = @medias.order(created_at: :desc)
-        render json: { status: 'SUCCESS', message: 'Loaded successfully', data: @medias }, status: :ok
+        medias = Media.where(account_id: params[:account_id])
+        #@artists = Artist.where("name RLIKE ?", "^#{filter_letter}")
+        medias = medias.where("asset_id RLIKE ?", "#{params[:asset_id]}") if params[:asset_id].present?
```

Comment:
what happens when we have special char like `$` in title?

Reply:
Added that in next commit of this PR